frontend/src/components/filter/FilterMenu.js

```diff
@@ -183,6 +183,7 @@ export default function FilterMenu({
       canBlur={canBlur}
       AlternateActionButton={ClearAllButton}
       onOpen={onOpen}
+      filterCount={filters.length}
```

Comment:
Can we update the `buttonText` prop adding the number there? Doing it that way means we don't have to pass a prop down to the dropdown menu and can keep the dropdown menu more generic (not having to worry about filters, even though I see a `test-id` that references a filter that was in there before this PR)

Reply:
Updated to use button text.